presto-orc/src/main/java/com/facebook/presto/orc/stream/LongInputStream.java

```diff
@@ -28,9 +28,15 @@ long next()
     void nextIntVector(int items, int[] vector)
             throws IOException;
 
+    void nextIntVector(int items, int[] vector, int offset)
+            throws IOException;
+
     void nextIntVector(int items, int[] vector, boolean[] isNull)
             throws IOException;
 
+    void nextIntVector(int items, int[] vector, int vectorOffset, boolean[] isNull, int isNullOffset)
```

Comment:
Remove `isNullOffset` since it's always 0. Argument for symmetricity is not a strong one here because ideally the signature should be `(boolean[] isNull, int[] vector, int offset, int length)`